tests/test_callback.py

```diff
@@ -0,0 +1,37 @@
+import os
+import re
+import ansible_runner
+
+
+def run_playbook(tmpdir):
+    os.environ['ANSIBLE_STDOUT_CALLBACK'] = "foreman"
+    os.environ['ANSIBLE_LOAD_CALLBACK_PLUGINS'] = "1"
+    # No connection is actually performed during the test
+    os.environ['FOREMAN_URL'] = "http://localhost"
+    os.environ['FOREMAN_SSL_CERT'] = "/dev/zero"
+    os.environ['FOREMAN_SSL_KEY'] = "/dev/zero"
+    os.environ['FOREMAN_DIR_STORE'] = str(tmpdir)
+    kwargs = {}
+    kwargs['playbook'] = os.path.join(os.getcwd(), 'tests', 'callback', 'three_hosts.yml')
+    kwargs['inventory'] = os.path.join(os.getcwd(), 'tests', 'callback', 'three_hosts')
+    kwargs['verbosity'] = 4
+    return ansible_runner.run(**kwargs)
+
+
+def test_callback(tmpdir, vcrmode):
+    run = run_playbook(tmpdir)
+    assert run.rc == 0
+    for file in os.listdir(str(tmpdir)):
```

Comment:
this means that if nothing was written, no assertions happen.

Reply:
Is that a bad assumption?

Reply:
I mean, FOREMAN_DIR_STORE variable makes Ansible to always write results.

Reply:
Well, I managed to break it, and wondered why tests were still green ;)

Reply:
more exactly, the folder is also empty when I run your code verbatim, so the asserts never trigger

Reply:
OMG I built a car without wheels. Lemme fix this.